target/linux/ipq40xx/image/Makefile

```diff
@@ -198,6 +198,21 @@ define Device/cilab_meshpoint-one
 endef
 TARGET_DEVICES += cilab_meshpoint-one
 
+define Device/xiaomi_meshd01
```

Comment:
As discussed in the DTS, you will have to adjust this to the new compatible, just with "_" instead of ",".